src/utils/combineReducers.js

```diff
@@ -59,19 +59,17 @@ function assertReducerSanity(reducers) {
     if (typeof initialState === 'undefined') {
       throw new Error(
         `Reducer "${key}" returned undefined during initialization. ` +
-        `If the state passed to the reducer is undefined, you must ` +
-        `explicitly return the initial state. The initial state may ` +
-        `not be undefined.`
+        `Reducers should never return undefined. Make sure this reducer ` +
+        `has a catch-all clause for unknown action types and that it returns a ` +
+        `default initial state if the state passed to it is undefined.`
       )
```

Comment:
I changed the error message here because if the user doesn't have a `default` case in their code, this is the error that will be thrown.